src/planner/dart/ConfigurationToConfiguration_to_ConfigurationToTSR.cpp

```diff
@@ -57,30 +64,27 @@ ConfigurationToConfiguration_to_ConfigurationToTSR::plan(
   auto ik = InverseKinematics::create(endEffectorBodyNode);
   ik->setDofs(mMetaSkeleton->getDofs());
 
-  // Convert TSR constraint into IK constraint
+  // Get the start state form the MetaSkeleton, since this is a DART planner.
+  auto startState = mMetaSkeletonStateSpace->createState();
+  mMetaSkeletonStateSpace->getState(mMetaSkeleton.get(), startState);
+
+  auto rng = std::move(cloneRNGFrom(*mDelegate->getRng())[0]);
```

Comment:
I could be wrong, but it seems `std::move()` isn't necessary.

Reply:
It appears this is needed, since this is a unique pointer and it can't be assigned, only moved.